pkg/operator/nfs/controller.go

```diff
@@ -455,7 +456,7 @@ func validateAccessMode(mode string) error {
 	switch s.ToLower(mode) {
 	case "readonly":
 	case "readwrite":
-	case "none":
+	case noneMode:
```

Comment:
if creating a `noneMode` should we also define a const for readonly and readwrite?

Reply:
Sure, I'll add that.